nvdaAPIVersions.json

```
@@ -206,5 +206,30 @@
 			"minor": 1,
 			"patch": 0
 		}
+	},
+	{
+		"description": "NVDA 2022.2.1",
+		"apiVer": {
+			"major": 2022,
+			"minor": 2,
+			"patch": 1
+		},
+		"backCompatTo": {
+			"major": 2022,
+			"minor": 1,
+			"patch": 0
+		},
+	{
```

fix up missing brace

```suggestion
		},
	},
	{
```